earth_enterprise/integration_test/step_impl/genewimageryresource.py

```diff
@@ -0,0 +1,77 @@
+from getgauge.python import step, before_scenario, Messages
+import os, random, fnmatch, subprocess, datetime
+from subprocess import Popen
+import shutil
+
+# Re-use some of the functionalities
+import assets
+
+# ------------------------
+# Helpful Not-Step Methods
+# ------------------------
+def doesKhassetFileExist(sPathUntilDotKiasset):
+   return os.path.exists(sPathUntilDotKiasset + ".kiasset/khasset.xml" )
+
+def executeCommand(sTestDir):
+   sImageryRoot = "/opt/google/share/tutorials/fusion/";
+   sImageryFile = "Imagery/bluemarble_4km.tif"
+   
+   # What if the Imagery wasn't downloaded
```

Comment:
minor: consider re-wording the comment to some thing like `Download the imagery if not available`